### DISCUSSION TITLE
go 1.22 无法适配win7

### DISCUSSION THREAD

**Discussion by 1062497537** [moonD4rk/HackBrowserData#416] [Q&A]
cmd .exe /c go build
..\..\browserdata\bookmark\bookmark.go:5:2: package log/slog is not in GOROOT (D:\src\log\slog)
note: imported by a module that requires go 1.22
..\..\browser\chromium\chromium.go:8:2: package slices is not in GOROOT (D:\src\slices)
note: imported by a module that requires go 1.22
hackbrowser编译需要go 1.22以上版本 

升级完go版本后确实可以解决该问题，但go 1.21版本后不再支持 Windows7、Windows2008、Windows2012

**Comment by 1062497537** [comment]
这就导致hackbrowser 在win7上报
![图片](https://github.com/user-attachments/assets/7f4607fd-9c97-4e2a-8dd4-38874a699d9d)

**Comment by moonD4rk** [comment]
This is a known issue, you can refer to the Link below and their corresponding solutions, we will consider lowering the golang version to address more usage scenarios.

- https://github.com/moonD4rk/HackBrowserData/issues/378
